docs/tutorials/medical/download_and_split_ixi.py

```diff
@@ -9,8 +9,8 @@
 import pandas as pd
 from sklearn.model_selection import train_test_split
 
-from fedbiomed.node.config import NodeConfig 
-from fedbiomed.node.environ import environ
+from fedbiomed.node.config import NodeConfig
+from fedbiomed.node.config import config
```

Comment:
```suggestion
#from fedbiomed.node.config import config
```

Remove this line ? 
`fedbiomed.node.config.config` does not exist

Apply same modification to `notebooks/medicalimage-segmentation/medical-image-segmentation-custom-unet-code.ipynb`

Reply:
Hi @mvesin,  I removed this line form the python script `download_and_split_ixi.py` but I couldn't find it in `notebooks/medicalimage-segmentation/medical-image-segmentation-custom-unet-code.ipynb`. Do I miss something?